### PR TITLE
Added support for library and database versions as well as for languages

### PR DESCRIPTION
**Pull request by martinslota**

### CODE REVIEW

File: src/test/generate-sample.pl [hunk 1]
```diff
@@ -21,11 +23,11 @@
...
-	subdivisions => [ { iso_code => 'Region' } ],
-	location => { latitude => 1.2, longitude => 3.4 },
+        country => { iso_code => 'Country', names => { en => 'Atlantis' } },
+        subdivisions => [ { iso_code => 'Region' } ],
+        location => { latitude => 1.2, longitude => 3.4 },
     },
 );
 
-open my $fh, '>:raw', 't.mmdb';
+open my $fh, '>:raw', 'sample.mmdb';
```
Comment (Leonidas-from-XIV) [inline src/test/generate-sample.pl]: What is the policy on this file? Is it meant to be included in the repo or not? I am a bit confused.

Comment (martinslota) [reply src/test/generate-sample.pl]: Don't think we have a policy.

The Perl code is super useful to see how the sample was generated. The sample itself is used in unit tests.

Comment (Leonidas-from-XIV) [reply src/test/generate-sample.pl]: Personally I think it should be included to not require Perl and stuff, so it seems this PR goes in the right direction.

Comment (martinslota) [reply src/test/generate-sample.pl]: Yeah well the Perl toolchain is a real pain to get up and running as well. I really suffered to get it up on my computer...

Comment (martinslota) [reply src/test/generate-sample.pl]: Note that the file was included before as well.

Comment (Leonidas-from-XIV) [reply src/test/generate-sample.pl]: Odd, I see it as added (not modified) in this PR.

Comment (martinslota) [reply src/test/generate-sample.pl]: The UI is just confusing.
Screenshot 2019-05-21 at 14 24 11

**Pull request merged**